monai/metrics/active_learning_metrics.py

```diff
@@ -123,3 +165,53 @@ def compute_variance(
         elif scalar_reduction == "sum":
             var_sum = torch.sum(variance)
             return var_sum
+
+
+def label_quality_score(
+    y_pred: torch.Tensor,
+    y: torch.Tensor,
+    include_background: bool = True,
+    spatial_map: bool = False,
+    scalar_reduction: str = "mean",
```

Comment:
could you please simpilify the function API by combining `spatial_map` boolean value and `scalar_reduction` into a single `reduction` arg? like these options: https://github.com/Project-MONAI/MONAI/blob/f4b65445514d0f8fc82ad7a10f54b1b5434ce7e7/monai/metrics/regression.py#L208-L210

Reply:
@wyli 

Done as suggested